src/main/java/org/sonar/plugins/clover/CloverSensor.java

```diff
@@ -48,24 +52,24 @@ public boolean shouldExecuteOnProject(Project project) {
 
   @Override
   public void analyse(Project project, SensorContext context) {
-    File report = getReportFromProperty(project);
+    File report = getReportFromProperty();
     if (reportExists(report)) {
-      factory.create(project, context).collect(report);
+      new CloverXmlReportParser(context, new InputFileProvider(fs)).collect(report);
     } else {
       LoggerFactory.getLogger(getClass()).warn("Clover XML report not found");
     }
   }
 
-  private File getReportFromProperty(Project project) {
+  private File getReportFromProperty() {
     String path = settings.getString(REPORT_PATH_PROPERTY);
     if (StringUtils.isNotEmpty(path)) {
-      return project.getFileSystem().resolvePath(path);
+        return pathResolver.relativeFile(fs.baseDir(), path);
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Make this line start at column 7. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://dory.sonarsource.com/coding_rules#rule_key=squid%3AIndentationCheck)